server/src/test/java/org/opensearch/index/IndexServiceTestUtils.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.index;
+
+import org.opensearch.common.unit.TimeValue;
+import org.opensearch.common.util.concurrent.AbstractAsyncTask;
+
+public final class IndexServiceTestUtils {
```

Comment:
Trading cleaner APIs to dedicated test helper class (to access package private methods)